dashboard/src/shared/Auth.ts

```diff
@@ -42,7 +42,9 @@ export class Auth {
       return [];
     }
     return [
-      "base64url.bearer.authorization.k8s.io." + btoa(token).replace(/=*$/g, ""),
+      // Trimming the b64 padding character ("=") as it is not accepted by k8s
+      // https://github.com/kubernetes/apiserver/blob/master/pkg/authentication/request/websocket/protocol.go#L38
+      "base64url.bearer.authorization.k8s.io." + btoa(token).replaceAll("=", ""),
```

Comment:
I was wondering why you weren't just removing them from the end, but given that `=` isn't part of radix-64, they can only be present as padding.

Reply:
Yep, it won't appear if the string is a multiple of a 3 character number since b64 encodes each 3 bytes. There is a good answer here: https://stackoverflow.com/questions/6916805/why-does-a-base64-encoded-string-have-an-sign-at-the-end
As you said, "=" can only be padding, so it is safe to be wiped out.